code-scanning/sbt-dependency-graph.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# More details at https://github.com/scalacenter/sbt-dependency-graph-action 
```

Comment:
Please add more details around what workflow does, how it does it and any pre-requisite steps needed for running this workflow.